ludwig/schema/llms/peft.py

```diff
@@ -69,6 +69,17 @@ class LoraConfig(BaseAdapterConfig):
         description="Bias type for Lora.",
     )
 
+    pretrained_weights: Optional[str] = schema_utils.String(
+        default=None, description="Path to pretrained weights for Lora.", allow_none=True
+    )
+
+    target_modules: Optional[list] = schema_utils.List(
+        str,
+        default=None,
+        allow_none=True,
+        description="List of modules to apply Lora to. If None, apply to all modules.",
+    )
```

Comment:
Is this needed?

Reply:
I recall this causing an error if this wasn't set.

Reply:
If I recall correctly, there was an error involving target_modules not being a parameter of a LoraConfig.